workflows/config/snakemake/config.yaml

```diff
@@ -32,3 +32,4 @@
 "poly_dfe_exec": "ext/polyDFE/polyDFE-2.0-linux-64-bit"
 "msmc_exec" : "ext/msmc/build/msmc"
 "stairwayplot_code" : "stairwayplot/swarmops.jar"
+"gone_code" : "GONE/Linux"
```

Comment:
does it only work with linux machine?

Reply:
linux or macos. Think that should suffice?